doc/devel/coding_guide.rst

```diff
@@ -182,6 +61,12 @@ PR Review guidelines
   'approve review').  If you do the merge please removed the
   ``'[MRG+N']`` prefix.
 
+* Make sure the Travis tests are passing before merging.
+
+  - The Travis tests automatically test on all of the Python versions
+    matplotlib supports whenever a pull request is created or updated.
+    The `tox` support in matplotlib may be useful for testing locally.
```

Comment:
Not entirely sure… I think this is some copy paste from the old contribution guidelines.
I think at some point we should go through this document again to make sure everything is up to date.